h/views/main.py

```diff
@@ -57,14 +57,24 @@ def robots(context, request):
 
 @view_config(route_name='stream')
 def stream(context, request):
-    atom = request.route_url('stream_atom')
-    rss = request.route_url('stream_rss')
-    return render_app(request, {
-        'link_tags': [
-            {'rel': 'alternate', 'href': atom, 'type': 'application/atom+xml'},
-            {'rel': 'alternate', 'href': rss, 'type': 'application/rss+xml'},
-        ]
-    })
+    if request.feature('search_page'):
+        q = request.params['q'].split(':')
+        if q[0] == 'tag':
```

Comment:
Similar to above, if the "q" parameter exists but without a colon then this will break.